jwst/resample/gwcs_drizzle.py

```diff
@@ -79,7 +79,7 @@ def __init__(self, product, outwcs=None, wt_scl=None,
 
         out_units = "cps"
 
-        self.outexptime = product.meta.resample.product_exposure_time or 0.0
+        self.outexptime = product.meta.exposure.effective_exposure_time or 0.0
```

Comment:
Where/how is `outexptime` used anywhere? If it's used in any of the drizzling computations, I think we should be using the new measurement time, not the old effective exposure time. If it's only used to pass a meta value to the output model/file being created, I'm not sure we need it anymore (because the output will have all the results from model blender).

Reply:
I think you're right it should be measurement time, sorry about that.  This is used to change the units from cts to cts/s in line 106.  It doesn't appear to be used anywhere else - those are the only two instances of that variable name in the entire code base.

Reply:
Agreed. It looks like the `outexptime` value is only ever used if `out_units = "counts"` and since we have `out_units` hardwired to "cps", the `outexptime` never gets used (at least for JWST data). So it doesn't actually matter what value we set it to, but having it set to TMEASURE is good for consistency.